digits/task.py

```diff
@@ -133,8 +133,8 @@ def path(self, filename, relative=False):
             path = filename
         else:
             path = os.path.join(self.job_dir, filename)
-        if relative:
-            path = os.path.relpath(path, config_value('jobs_dir'))
+            if relative:
+                path = os.path.relpath(path, config_value('jobs_dir'))
```

Comment:
unrelated with this change but that code duplication makes it look like `Task` and `Job` classes could share a common ancestor